src/Workspaces/Core/MSBuild/MSBuild/MSBuildProjectLoader.Worker.cs

```diff
@@ -212,10 +212,8 @@ private async Task<ImmutableArray<ProjectFileInfo>> LoadProjectFileInfosAsync(st
                 foreach (var projectFileInfo in projectFileInfos)
                 {
                     // If any diagnostics were logged during build, we'll carry on and try to produce a meaningful project.
-                    if (!projectFileInfo.Log.IsEmpty)
-                    {
-                        _diagnosticReporter.Report(projectFileInfo.Log);
-                    }
+                    // Note: any diagnostics would have been logged to the original project file's log.
```

Comment:
should probably trigger an /azp run here - I don't think the single machine tests (that run msbuild workspace tests) generally run on PRs?